feature/manager_internal_test.go

```diff
@@ -104,6 +110,10 @@ func testManager(t *testing.T, test managerTest) {
 		if test.cfg.NoStaticRemoteKey {
 			assertUnset(lnwire.StaticRemoteKeyOptional)
 		}
+		if test.cfg.NoAnchors {
```

Comment:
I double checked this and I think `ScriptEnforcedLeaseRequired` and `ScriptEnforcedLeaseOptional` are disabled by default, so this test passes even without your fix.